tests/unit/legacy/api/test_json.py

```diff
@@ -249,7 +247,7 @@ def test_detail_renders(self, pyramid_config, db_request, db_session):
         expected_urls = []
         for project_url in reversed(project_urls):
             expected_urls.append(tuple(project_url.split(",")))
-        expected_urls = OrderedDict(tuple(expected_urls))
+        expected_urls = dict(tuple(expected_urls))
```

Comment:
I think the order matters here, did we lose the implicit ordering in the DB?

Reply:
seems we still have implicit ordering... but it's not reversed? see a112d01b